tools/filter_by_fasta_ids/filter_by_fasta_ids.xml

```diff
@@ -51,6 +54,24 @@ $dedup
             <when value="" />
             <when value="id_list">
                 <param name="identifiers" type="data" format="txt" label="List of IDs to extract sequences for"/>
+                
+                
+                <conditional name="id_regex">
+                    <param name="find" type="select" label="Match IDs by">
+                        <option value="beginning">ID is expected at the beginning: &gt;ID</option>
+                        <option value="pattern">regex pattern</option>
+                    </param>
+                    <when value="beginning"/>
+                    <when value="pattern">
+                        <param name="pattern" type="text" value="" label="regex search pattern for ID">
```

Comment:
The default value in the python script is `>([^| ]+)` isn't it? Maybe we should add this here or mention it in the help of this param.

Reply:
Modified according to your suggestion